server/src/StatusSender.ts

```diff
@@ -0,0 +1,61 @@
+import { StatusNotification } from 'dbt-language-server-common';
+import { FileChangeType, _Connection } from 'vscode-languageserver';
+import { FeatureFinder } from './FeatureFinder';
+import { FileChangeListener } from './FileChangeListener';
+
+export class StatusSender {
+  constructor(
+    private connection: _Connection,
+    private projectPath: string,
+    private featureFinder: FeatureFinder,
+    private fileChangeListener: FileChangeListener,
+  ) {
+    this.fileChangeListener.onDbtPackagesYmlChanged(e => this.onDbtPackagesYmlChanged(e));
+    this.featureFinder.packagesYmlExistsPromise
+      .then(packagesYmlFound => this.sendPackagesStatus(packagesYmlFound))
+      .catch(e => console.log(`Error while finding packages.yml: ${e instanceof Error ? e.message : String(e)}`));
+  }
+
+  sendStatus(): void {
+    const statusNotification: StatusNotification = {
+      projectPath: this.projectPath,
+      pythonStatus: {
+        path: this.featureFinder.getPythonPath(),
+      },
+      dbtStatus: {
+        versionInfo: this.featureFinder.versionInfo,
+      },
```

Comment:
Why we do not send `packagesStatus` ?

Reply:
We send it separately from `sendPackagesStatus` method. Not sure that we need duplicate.